frontend/src/components/App/Settings/SettingsCluster.tsx

```diff
@@ -327,6 +327,27 @@ export default function SettingsCluster() {
     );
   }
 
+  // Display the original name of the cluster if it was loaded from a kubeconfig file.
```

Comment:
The commit message is pretty good.

I think adding the component into the context part will make it easier to understand when reading with `git log --oneline`.

`frontend:  SettingsCluster: Add original name to cluster settings view for non dynamic clusters`